integration-cli/docker_cli_events_test.go

```diff
@@ -140,6 +140,35 @@ func (s *DockerSuite) TestEventsContainerEvents(c *check.C) {
 	c.Assert(containerEvents[4], checker.Equals, "destroy", check.Commentf(out))
 }
 
+func (s *DockerSuite) TestEventsContainerEventsAttrSort(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
Shouldn't be a need for this testRequires following merge of https://github.com/docker/docker/pull/19839